lib/type-registry.js

```diff
@@ -32,6 +32,10 @@ TypeRegistry.prototype.registerType = function(typeName, converter) {
 
   typeName = typeName.toLowerCase();
 
+  if (typeName === 'file') {
+    throw new Error('Cannot override built-in "file" type.');
```

Comment:
> Also, I don't see it being caught anywhere. So, I am assuming that it will bubble up and eventually handled. Correct?

It will bubble up and crash the app at startup, which is my intent. Let the user know about the problem immediately at startup, rather than later when some of their requests start failing.